README.md

```diff
@@ -120,6 +129,7 @@ Done! You can change not only the landing page but any pages like this way. (You
   * `templates` - HTML files used as ejs template. `publish` script converts a markdown file to HTML based on templates under this directory.
 * `dist` - Files compiled by `build` script. `deploy` script deploys a website to GitHub pages based on this directory. Do not change the files under this directory directly.
 * `server` - Files compiled by `build` script. `start` script opens local server based on this directory. Do not change the files under this directory directly.
+* `doc-translation` - The folder contains documentation in several languages written in markdown format.
```

Comment:
```suggestion
```

Please remove this line from README.md and README-IND.md

Reply:
@parksb Sorry for being sleepy I forgot to remove the doc-translation line, thanks for reminding me

I've finished removing the `doc-translation` line and changing the link which will display the `README-IND.md` file